proto/cosmos/protocolpool/v1/tx.proto

```diff
@@ -29,6 +29,12 @@ service Msg {
 
   // ClaimBudget defines a method to claim the distributed budget.
   rpc ClaimBudget(MsgClaimBudget) returns (MsgClaimBudgetResponse);
+
+  // AddContinuousFunds defines a method to add funds continuously.
+  rpc AddContinuousFunds(MsgAddContinuousFunds) returns (MsgAddContinuousFundsResponse);
```

Comment:
```suggestion
  rpc AddContinuousFund(MsgAddContinuousFund) returns (MsgAddContinuousFundResponse);
```

since the user would only be adding a single fund and not plural

Reply:
Also `Add` doesn't sound right, we should probably use `CreateContinuousFund`.